### PR TITLE
add 'verified by' to contact profiles

### LINKED ISSUE #2468: add "verified by" to profile
from nov22 consistency discussions, we want to have the information "verified by ..." at least in the contact profiles.

since https://github.com/deltachat/deltachat-core-rust/pull/3839 these information are available in the ffi as [`dc_contact_get_verifier_id()`](https://c.delta.chat/classdc__contact__t.html#a8de31ca4093219fafcb086e712e2acd5) and [`dc_contact_get_verifier_addr()`](https://c.delta.chat/classdc__contact__t.html#a4f10fbe48e55d237c4673fbb141506b8).

related to https://github.com/deltachat/deltachat-android/issues/2099

once this is settled on android and strings are added, we should file corresponding issues for ios/desktop.

### PR DESCRIPTION
**Pull request by r10s**
- [x] add new APIs
- [x] add basic verified-by line
- [x] add an icon
- [x] handle tap (i think, open verifier contact)
- [x] cleanup layout: move "last seen" to separate line, add "info" headline, make "send message" button-blue

closes #2468

### CODE REVIEW

File: src/org/thoughtcrime/securesms/ProfileSettingsFragment.java [hunk 1]
```diff
@@ -233,6 +236,18 @@ public void onSharedChatClicked(int chatId) {
     getActivity().finish();
   }
 
+  private void onVerifiedByClicked() {
+    DcContact dcContact = dcContext.getContact(contactId);
+    if (dcContact.isVerified()) {
+      int verifierId = dcContact.getVerifierId();
+      if (verifierId != 0 && verifierId != contactId) {
```
Comment (adbenitez) [inline src/org/thoughtcrime/securesms/ProfileSettingsFragment.java:243]: if this gets fixed in core then here we need to change and check if verifierId == SELF_CONTACT_ID

File: src/org/thoughtcrime/securesms/ProfileSettingsItem.java [hunk 2]
```diff
@@ -23,7 +27,15 @@ protected void onFinishInflate() {
...
-  public void set(String label) {
+  public void set(String label, int labelColor, int iconLeft) {
     labelView.setText(label==null? "" : label);
+    labelView.setCompoundDrawablesWithIntrinsicBounds(iconLeft, 0,0,0);
+
+    // we need different color getters as `labelColor` is `R.color.name` while default is `R.attr.name`
+    if (labelColor != 0) {
+      labelView.setTextColor(ContextCompat.getColor(getContext(), labelColor));
+    } else {
+      labelView.setTextColor(ResUtil.getColor(getContext(), R.attr.emoji_text_color)); //
```
Comment (adbenitez) [inline src/org/thoughtcrime/securesms/ProfileSettingsItem.java:38]: there is an empty comment at the end

**Review by adbenitez — APPROVED** [review]

File: src/org/thoughtcrime/securesms/ProfileSettingsItem.java (hunk 2 above)
Comment (r10s) [reply src/org/thoughtcrime/securesms/ProfileSettingsItem.java]: removed

File: src/org/thoughtcrime/securesms/ProfileSettingsFragment.java (hunk 1 above)
Comment (r10s) [reply src/org/thoughtcrime/securesms/ProfileSettingsFragment.java:243]: let's see what the outcome of https://github.com/deltachat/deltachat-core-rust/issues/4748 will be - if we did sth. wrong or there will be a fix in core. currently we seem to have the same state as desktop, so i think, we can merge this pr in and iterate over in a subsequent pr as needed

**Pull request merged**